packages/components/src/components/list-item-location/list-item-location.tsx

```diff
@@ -80,8 +80,13 @@ export class ListItemLocation {
     }
 
     componentWillLoad(): void {
-        this.iconURLToRender = this.icon ? this.icon : this.location?.properties.imageURL;
-        this.updateBadge();
+        // Google default icon
+        if (this.location.properties.type === 'google_places') {
```

Comment:
Could it make sense to add the SVG from outside of this component (using the icon prop) to avoid having Google-specifics in this component? 🤔 I'm not sure.

Reply:
Good idea. Refactored the usage of the icon on google places. ✅